lib/node_modules/@stdlib/string/to-well-formed/examples/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/* eslint-disable no-new-wrappers */
+
+'use strict';
+
+var str2wellformed = require( './../lib' );
+
+console.log( str2wellformed( '' ) );
+
```

Comment:
```suggestion
console.log( str2wellformed( '' ) );
// => ''

```

I will prefer to do it for all console.log here